ITP_JAN25| Sheetal_Kharab | Module_Data_flows| Book_Library

sheetalkharab opened the pull request:
<!--

You must title your PR like this:

COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK

For example,

NW4 | Carol Owen | HTML-CSS-Module | Week1

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/contributing/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

find some bug and update code and fix delete button

## Questions

Ask any questions you have for your reviewer.

shieldo reviewed (approved):
Well done @sheetalkharab - you've made all the fixes here so the script runs without errors and doesn't have the buggy behaviour that it started with - everything listed is fixed!

sheetalkharab commented:
Hello @shieldo, Thank you for review. Sorry, I changed the label without asking. so I just removed it and please suggest, can I change label to 'complete' .

shieldo commented:
Hello! Yes, it's complete and you can!

On Sun, 6 Apr 2025, 22:24 sheetalkharab, ***@***.***> wrote:

> Hello @shieldo <https://github.com/shieldo>, Thank you for review. Sorry,
> I changed the label without asking. so I just removed it and please
> suggest, can I change label to 'complete' .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Data-Flows/pull/170#issuecomment-2781675476>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAXYAE7C4UJ6ALTP3ZYATT2YGLPZAVCNFSM6AAAAABZXTAVCWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOBRGY3TKNBXGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: sheetalkharab]*sheetalkharab* left a comment
> (CodeYourFuture/Module-Data-Flows#170)
> <https://github.com/CodeYourFuture/Module-Data-Flows/pull/170#issuecomment-2781675476>
>
> Hello @shieldo <https://github.com/shieldo>, Thank you for review. Sorry,
> I changed the label without asking. so I just removed it and please
> suggest, can I change label to 'complete' .
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Data-Flows/pull/170#issuecomment-2781675476>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAXYAE7C4UJ6ALTP3ZYATT2YGLPZAVCNFSM6AAAAABZXTAVCWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOBRGY3TKNBXGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sheetalkharab commented:
@shieldo ,Thank you
